### PR TITLE
Updated Template to install latest version of DSCForLinux extension

### PR DESCRIPTION
**Pull request by balukambala**
…in the extension properties

### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

* Updated Template to install latest version of DSCForLinux extension
* Added autoUpgradeMinorVersion field in the extension settings

### CODE REVIEW

File: 201-dsc-linux-azure-storage-on-ubuntu/metadata.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "itemDisplayName": "DSCForLinux extension on a Ubuntu VM",
-  "description": "This template creates a Ubuntu VM and installs the DSCForLinux extension 2.0",
-  "summary": "This template creates a Ubuntu VM and installs the DSCForLinux extension 2.0",
-  "githubUsername": "lizzha",
-  "dateUpdated": "2016-03-15"
+  "description": "This template creates a Ubuntu VM and installs the DSCForLinux extension 2.70 or latest",
+  "summary": "This template creates a Ubuntu VM and installs the DSCForLinux extension 2.70 or latest",
+  "githubUsername": "balukambala",
```
Comment (bmoore-msft) [inline 201-dsc-linux-azure-storage-on-ubuntu/metadata.json]: did you want to take ownership of these?

**Comment by bmoore-msft** [comment]
Can you take a look at the build failures?

File: 201-dsc-linux-azure-storage-on-ubuntu/metadata.json (hunk 1 above)
Comment (balukambala) [reply 201-dsc-linux-azure-storage-on-ubuntu/metadata.json]: yes, My manager take over the Linux extension for DSC. I am working on enhancements on DSC Linux extension. I am going to make few more changes to these templates.

Comment (bmoore-msft) [reply 201-dsc-linux-azure-storage-on-ubuntu/metadata.json]: Cool - can you look at the build failures - it looks like the param file is not using generated values and the artifacts are not using BPs for the repo...

Comment (balukambala) [reply 201-dsc-linux-azure-storage-on-ubuntu/metadata.json]: Param file has fixed values and I see this error in the build output:

"error": "Error: Command failed: /bin/sh -c az group deployment create  --resource-group qstci-78841b45-b6ce-e804-fc02-14b048f55cf7 --template-file temp/435d7257-fdea-3600-79db-4cf530ccd4d4 --parameters temp/903447eb-50a6-1f1d-62a6-cfe380add187 -o json\nERROR: Deployment failed. {\r\n  \"error\": {\r\n    \"code\": \"StorageAccountAlreadyTaken\",\r\n    \"message\": \"The storage account named dscuniqueaccount1 is already taken.\"\r\n  }\r\n}\n"

It looks like storage account is already exist under this resource group. Are you expecting me to make these parameters dynamic?

Comment (bmoore-msft) [reply 201-dsc-linux-azure-storage-on-ubuntu/metadata.json]: Yes, see the contribution guide for details...

Comment (balukambala) [reply 201-dsc-linux-azure-storage-on-ubuntu/metadata.json]: Build is successful. Please let me know if there is any pending here.

**Pull request merged**